.github/workflows/tscheck.yml

```
@@ -0,0 +1,33 @@
+name: Lint
```

```suggestion
name: Typescript Check 
```

Maybe you should merge Lint and typescript check